support-payment-api/src/main/scala/backend/AmazonPayBackend.scala

```diff
@@ -43,7 +43,9 @@ class AmazonPayBackend(
     with PaymentBackend {
 
   private def amazonPayEnabled =
-    switchService.allSwitches.map(switch => switch.oneOffPaymentMethods.exists(s => s.switches.amazonPay.state.isOn))
+    switchService.allSwitches.map(switch =>
+      switch.oneOffPaymentMethods.exists(s => s.switches.amazonPay.exists(_.state.isOn)),
+    )
```

Comment:
```suggestion
 switchService.allSwitches.map(_.oneOffPaymentMethods.exists(_.switches.amazonPay.exists(_.state.isOn)))
```
Very minor, maybe this is slightly easier to parse?